[2023-09-14T14:38:07.901+0000] {processor.py:156} INFO - Started process (PID=18687) to work on /opt/airflow/dags/test.py
[2023-09-14T14:38:07.903+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:38:07.905+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:07.904+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:38:08.902+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:38:09.676+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.667+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:keren
[2023-09-14T14:38:09.735+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.734+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:keren
[2023-09-14T14:38:09.769+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.769+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:keren
[2023-09-14T14:38:09.772+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.772+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:38:09.838+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.837+0000] {dag.py:2747} INFO - Creating ORM DAG for keren
[2023-09-14T14:38:09.867+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:09.867+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:38:09.980+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.094 seconds
[2023-09-14T14:38:40.401+0000] {processor.py:156} INFO - Started process (PID=18747) to work on /opt/airflow/dags/test.py
[2023-09-14T14:38:40.407+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:38:40.409+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:40.408+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:38:41.102+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:38:41.460+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:41.459+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:38:41.636+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:41.635+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:38:41.777+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.381 seconds
[2023-09-14T14:38:42.737+0000] {processor.py:156} INFO - Started process (PID=18750) to work on /opt/airflow/dags/test.py
[2023-09-14T14:38:42.744+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:38:42.750+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:42.749+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:38:43.684+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:38:43.827+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:43.827+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:38:43.925+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:38:43.925+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:38:44.011+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.288 seconds
[2023-09-14T14:39:14.483+0000] {processor.py:156} INFO - Started process (PID=18807) to work on /opt/airflow/dags/test.py
[2023-09-14T14:39:14.486+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:39:14.497+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:14.497+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:39:15.272+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:39:15.470+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:15.470+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:39:15.600+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:15.598+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:39:15.689+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.213 seconds
[2023-09-14T14:39:45.883+0000] {processor.py:156} INFO - Started process (PID=18866) to work on /opt/airflow/dags/test.py
[2023-09-14T14:39:45.885+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:39:45.887+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:45.886+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:39:46.256+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:39:46.507+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:46.507+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:39:46.630+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:39:46.630+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:39:46.715+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.836 seconds
[2023-09-14T14:40:17.099+0000] {processor.py:156} INFO - Started process (PID=18923) to work on /opt/airflow/dags/test.py
[2023-09-14T14:40:17.112+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:40:17.114+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:17.114+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:40:19.975+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:40:20.175+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:20.175+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:40:20.286+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:20.285+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:40:20.400+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 3.307 seconds
[2023-09-14T14:40:50.558+0000] {processor.py:156} INFO - Started process (PID=18985) to work on /opt/airflow/dags/test.py
[2023-09-14T14:40:50.559+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:40:50.561+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:50.561+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:40:51.021+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:40:51.291+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:51.290+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:40:51.407+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:40:51.406+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:40:51.487+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.934 seconds
[2023-09-14T14:41:00.217+0000] {processor.py:156} INFO - Started process (PID=19011) to work on /opt/airflow/dags/test.py
[2023-09-14T14:41:00.218+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:41:00.220+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:00.220+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:41:01.006+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:41:01.036+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:01.035+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:41:01.144+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:01.144+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:41:01.240+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.039 seconds
[2023-09-14T14:41:31.924+0000] {processor.py:156} INFO - Started process (PID=19070) to work on /opt/airflow/dags/test.py
[2023-09-14T14:41:31.927+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:41:31.930+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:31.929+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:41:32.364+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:41:32.526+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:32.526+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:41:32.603+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:41:32.603+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:41:32.675+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.756 seconds
[2023-09-14T14:42:03.244+0000] {processor.py:156} INFO - Started process (PID=19127) to work on /opt/airflow/dags/test.py
[2023-09-14T14:42:03.256+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:42:03.257+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:03.257+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:42:03.724+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:42:03.993+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:03.992+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:42:04.089+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:04.089+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:42:04.154+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.915 seconds
[2023-09-14T14:42:17.185+0000] {processor.py:156} INFO - Started process (PID=19153) to work on /opt/airflow/dags/test.py
[2023-09-14T14:42:17.198+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:42:17.199+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:17.199+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:42:17.839+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:42:17.915+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:17.914+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:42:18.019+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:18.019+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:42:18.135+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.954 seconds
[2023-09-14T14:42:48.469+0000] {processor.py:156} INFO - Started process (PID=19210) to work on /opt/airflow/dags/test.py
[2023-09-14T14:42:48.471+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:42:48.472+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:48.472+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:42:49.338+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:42:49.487+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:49.487+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:42:49.561+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:42:49.561+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:42:49.632+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.167 seconds
[2023-09-14T14:43:19.783+0000] {processor.py:156} INFO - Started process (PID=19267) to work on /opt/airflow/dags/test.py
[2023-09-14T14:43:19.785+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:43:19.786+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:19.786+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:43:20.149+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:43:20.369+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:20.361+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:43:20.488+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:20.487+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:43:20.617+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.837 seconds
[2023-09-14T14:43:51.414+0000] {processor.py:156} INFO - Started process (PID=19324) to work on /opt/airflow/dags/test.py
[2023-09-14T14:43:51.417+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:43:51.425+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:51.419+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:43:51.864+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:43:52.097+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:52.097+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:43:52.171+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:43:52.171+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:43:52.248+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.838 seconds
[2023-09-14T14:44:22.318+0000] {processor.py:156} INFO - Started process (PID=19381) to work on /opt/airflow/dags/test.py
[2023-09-14T14:44:22.320+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:44:22.321+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:44:22.321+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:44:22.415+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:44:22.388+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd42880910>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42880910>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42880910>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:44:22.416+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:44:22.458+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.144 seconds
[2023-09-14T14:44:52.540+0000] {processor.py:156} INFO - Started process (PID=19438) to work on /opt/airflow/dags/test.py
[2023-09-14T14:44:52.542+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:44:52.543+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:44:52.543+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:44:52.762+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:44:52.739+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd3f34be90>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f34be90>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f34be90>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:44:52.764+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:44:52.833+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.297 seconds
[2023-09-14T14:45:23.467+0000] {processor.py:156} INFO - Started process (PID=19493) to work on /opt/airflow/dags/test.py
[2023-09-14T14:45:23.470+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:45:23.471+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:45:23.471+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:45:23.618+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:45:23.581+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd428d6ad0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd428d6ad0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd428d6ad0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:45:23.631+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:45:23.718+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.261 seconds
[2023-09-14T14:45:54.248+0000] {processor.py:156} INFO - Started process (PID=19549) to work on /opt/airflow/dags/test.py
[2023-09-14T14:45:54.250+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:45:54.251+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:45:54.251+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:45:54.364+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:45:54.347+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd42880ed0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42880ed0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42880ed0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:45:54.365+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:45:54.409+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.164 seconds
[2023-09-14T14:46:25.361+0000] {processor.py:156} INFO - Started process (PID=19606) to work on /opt/airflow/dags/test.py
[2023-09-14T14:46:25.368+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:46:25.376+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:46:25.376+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:46:25.493+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:46:25.474+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd3f330950>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f330950>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f330950>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:46:25.496+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:46:25.531+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.174 seconds
[2023-09-14T14:46:56.034+0000] {processor.py:156} INFO - Started process (PID=19663) to work on /opt/airflow/dags/test.py
[2023-09-14T14:46:56.036+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:46:56.037+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:46:56.037+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:46:56.246+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:46:56.188+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd42885ad0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42885ad0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd42885ad0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:46:56.247+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:46:56.296+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.265 seconds
[2023-09-14T14:47:26.677+0000] {processor.py:156} INFO - Started process (PID=19721) to work on /opt/airflow/dags/test.py
[2023-09-14T14:47:26.679+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:47:26.681+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:47:26.680+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:47:26.812+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:47:26.796+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fdd3f345d10>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 788, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f345d10>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test.py", line 3, in <module>
    from script.get_json import data_final
  File "/opt/airflow/dags/script/get_json.py", line 5, in <module>
    data_req = requests.get('https://api.coindesk.com/v1/bpi/currentprice.json')
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.coindesk.com', port=443): Max retries exceeded with url: /v1/bpi/currentprice.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdd3f345d10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2023-09-14T14:47:26.814+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:47:26.874+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.201 seconds
[2023-09-14T14:47:57.241+0000] {processor.py:156} INFO - Started process (PID=19779) to work on /opt/airflow/dags/test.py
[2023-09-14T14:47:57.243+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:47:57.244+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:47:57.244+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:47:57.778+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:47:57.962+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:47:57.962+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:47:58.044+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:47:58.043+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:47:58.125+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.892 seconds
[2023-09-14T14:48:28.227+0000] {processor.py:156} INFO - Started process (PID=19836) to work on /opt/airflow/dags/test.py
[2023-09-14T14:48:28.255+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:48:28.257+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:48:28.257+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:48:28.854+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:48:29.045+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:48:29.045+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:48:29.133+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:48:29.132+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:48:29.198+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.979 seconds
[2023-09-14T14:48:59.736+0000] {processor.py:156} INFO - Started process (PID=19892) to work on /opt/airflow/dags/test.py
[2023-09-14T14:48:59.737+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:48:59.739+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:48:59.738+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:49:00.273+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:49:00.447+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:49:00.447+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:49:00.528+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:49:00.528+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:49:00.601+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.869 seconds
[2023-09-14T14:49:30.673+0000] {processor.py:156} INFO - Started process (PID=19951) to work on /opt/airflow/dags/test.py
[2023-09-14T14:49:30.674+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:49:30.677+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:49:30.675+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:49:31.049+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:49:31.194+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:49:31.194+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:49:31.261+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:49:31.261+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:49:31.318+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.650 seconds
[2023-09-14T14:50:02.153+0000] {processor.py:156} INFO - Started process (PID=20008) to work on /opt/airflow/dags/test.py
[2023-09-14T14:50:02.155+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:50:02.156+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:02.156+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:50:02.509+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:50:02.699+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:02.698+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:50:02.773+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:02.773+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:50:02.831+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.681 seconds
[2023-09-14T14:50:33.118+0000] {processor.py:156} INFO - Started process (PID=20061) to work on /opt/airflow/dags/test.py
[2023-09-14T14:50:33.124+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:50:33.135+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:33.134+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:50:33.883+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:50:34.127+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:34.118+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:50:34.240+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:50:34.240+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:50:34.346+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.232 seconds
[2023-09-14T14:51:04.553+0000] {processor.py:156} INFO - Started process (PID=20118) to work on /opt/airflow/dags/test.py
[2023-09-14T14:51:04.554+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:51:04.564+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:04.555+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:51:04.918+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:51:05.054+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:05.053+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:51:05.119+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:05.118+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:51:05.180+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.631 seconds
[2023-09-14T14:51:35.857+0000] {processor.py:156} INFO - Started process (PID=20175) to work on /opt/airflow/dags/test.py
[2023-09-14T14:51:35.871+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:51:35.888+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:35.888+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:51:36.416+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:51:36.597+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:36.597+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:51:36.675+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:51:36.666+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:51:36.739+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.886 seconds
[2023-09-14T14:52:07.273+0000] {processor.py:156} INFO - Started process (PID=20234) to work on /opt/airflow/dags/test.py
[2023-09-14T14:52:07.285+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:52:07.295+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:07.294+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:52:08.720+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:52:08.969+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:08.969+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:52:09.090+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:09.090+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:52:09.178+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.912 seconds
[2023-09-14T14:52:39.402+0000] {processor.py:156} INFO - Started process (PID=20294) to work on /opt/airflow/dags/test.py
[2023-09-14T14:52:39.410+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:52:39.420+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:39.420+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:52:39.892+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:52:40.121+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:40.121+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:52:40.236+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:52:40.236+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:52:40.294+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.905 seconds
[2023-09-14T14:53:10.473+0000] {processor.py:156} INFO - Started process (PID=20352) to work on /opt/airflow/dags/test.py
[2023-09-14T14:53:10.475+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:53:10.477+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:10.476+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:53:11.182+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:53:11.357+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:11.356+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:53:11.420+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:11.420+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:53:11.484+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.015 seconds
[2023-09-14T14:53:41.904+0000] {processor.py:156} INFO - Started process (PID=20410) to work on /opt/airflow/dags/test.py
[2023-09-14T14:53:41.905+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:53:41.907+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:41.907+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:53:42.209+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:53:42.365+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:42.365+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:53:42.429+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:53:42.429+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:53:42.474+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.574 seconds
[2023-09-14T14:54:12.624+0000] {processor.py:156} INFO - Started process (PID=20467) to work on /opt/airflow/dags/test.py
[2023-09-14T14:54:12.625+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:54:12.627+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:12.626+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:54:13.180+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:54:13.369+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:13.369+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:54:13.469+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:13.468+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:54:13.601+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.983 seconds
[2023-09-14T14:54:44.301+0000] {processor.py:156} INFO - Started process (PID=20533) to work on /opt/airflow/dags/test.py
[2023-09-14T14:54:44.303+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:54:44.305+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:44.305+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:54:44.732+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:54:44.934+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:44.932+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:54:45.020+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:54:45.019+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:54:45.066+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.781 seconds
[2023-09-14T14:55:15.577+0000] {processor.py:156} INFO - Started process (PID=20590) to work on /opt/airflow/dags/test.py
[2023-09-14T14:55:15.578+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:55:15.579+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:15.579+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:55:16.078+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:55:16.376+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:16.376+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:55:16.518+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:16.517+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:55:16.657+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.091 seconds
[2023-09-14T14:55:47.236+0000] {processor.py:156} INFO - Started process (PID=20649) to work on /opt/airflow/dags/test.py
[2023-09-14T14:55:47.248+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:55:47.250+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:47.250+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:55:48.052+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:55:48.281+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:48.280+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:55:48.375+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:55:48.374+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:55:48.483+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.251 seconds
[2023-09-14T14:56:18.808+0000] {processor.py:156} INFO - Started process (PID=20707) to work on /opt/airflow/dags/test.py
[2023-09-14T14:56:18.818+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:56:18.820+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:18.820+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:56:19.306+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:56:19.486+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:19.486+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:56:19.572+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:19.572+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:56:19.639+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.835 seconds
[2023-09-14T14:56:49.985+0000] {processor.py:156} INFO - Started process (PID=20765) to work on /opt/airflow/dags/test.py
[2023-09-14T14:56:49.987+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:56:49.989+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:49.989+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:56:50.293+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:56:50.478+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:50.478+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:56:50.552+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:56:50.552+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:56:50.642+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.666 seconds
[2023-09-14T14:57:20.769+0000] {processor.py:156} INFO - Started process (PID=20821) to work on /opt/airflow/dags/test.py
[2023-09-14T14:57:20.773+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:57:20.783+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:20.782+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:57:21.154+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:57:21.307+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:21.307+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:57:21.390+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:21.390+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:57:21.489+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.731 seconds
[2023-09-14T14:57:51.602+0000] {processor.py:156} INFO - Started process (PID=20879) to work on /opt/airflow/dags/test.py
[2023-09-14T14:57:51.603+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:57:51.613+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:51.612+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:57:51.943+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:57:52.109+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:52.109+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:57:52.194+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:57:52.194+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:57:52.261+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.662 seconds
[2023-09-14T14:58:22.500+0000] {processor.py:156} INFO - Started process (PID=20937) to work on /opt/airflow/dags/test.py
[2023-09-14T14:58:22.515+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:58:22.535+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:22.534+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:58:23.020+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:58:23.267+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:23.266+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:58:23.384+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:23.384+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:58:23.474+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.990 seconds
[2023-09-14T14:58:53.721+0000] {processor.py:156} INFO - Started process (PID=20996) to work on /opt/airflow/dags/test.py
[2023-09-14T14:58:53.724+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:58:53.728+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:53.728+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:58:54.148+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:58:54.321+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:54.321+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:58:54.385+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:58:54.385+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:58:54.463+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.747 seconds
[2023-09-14T14:59:24.729+0000] {processor.py:156} INFO - Started process (PID=21053) to work on /opt/airflow/dags/test.py
[2023-09-14T14:59:24.731+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:59:24.732+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:24.732+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:59:25.076+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:59:25.267+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:25.267+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:59:25.356+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:25.356+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:59:25.423+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.698 seconds
[2023-09-14T14:59:55.589+0000] {processor.py:156} INFO - Started process (PID=21111) to work on /opt/airflow/dags/test.py
[2023-09-14T14:59:55.590+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T14:59:55.592+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:55.592+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T14:59:56.457+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T14:59:56.681+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:56.680+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T14:59:56.774+0000] {logging_mixin.py:149} INFO - [2023-09-14T14:59:56.774+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T15:00:00+00:00, run_after=2023-09-14T16:00:00+00:00
[2023-09-14T14:59:56.874+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.293 seconds
[2023-09-14T15:00:26.996+0000] {processor.py:156} INFO - Started process (PID=21171) to work on /opt/airflow/dags/test.py
[2023-09-14T15:00:26.997+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:00:27.008+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:27.008+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:00:27.553+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:00:27.775+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:27.774+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:00:27.867+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:27.867+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:00:27.980+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.989 seconds
[2023-09-14T15:00:58.379+0000] {processor.py:156} INFO - Started process (PID=21229) to work on /opt/airflow/dags/test.py
[2023-09-14T15:00:58.392+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:00:58.397+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:58.397+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:00:59.218+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:00:59.431+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:59.431+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:00:59.519+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:00:59.519+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:00:59.589+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.218 seconds
[2023-09-14T15:01:30.474+0000] {processor.py:156} INFO - Started process (PID=21288) to work on /opt/airflow/dags/test.py
[2023-09-14T15:01:30.476+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:01:30.493+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:01:30.493+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:01:31.461+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:01:31.692+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:01:31.692+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:01:31.774+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:01:31.774+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:01:31.852+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.390 seconds
[2023-09-14T15:02:02.056+0000] {processor.py:156} INFO - Started process (PID=21351) to work on /opt/airflow/dags/test.py
[2023-09-14T15:02:02.057+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:02:02.059+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:02.058+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:02:02.355+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:02:02.504+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:02.503+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:02:02.557+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:02.557+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:02:02.619+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.567 seconds
[2023-09-14T15:02:32.709+0000] {processor.py:156} INFO - Started process (PID=21407) to work on /opt/airflow/dags/test.py
[2023-09-14T15:02:32.711+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:02:32.713+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:32.713+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:02:33.460+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:02:33.633+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:33.633+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:02:33.696+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:02:33.696+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:02:33.806+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.101 seconds
[2023-09-14T15:03:04.395+0000] {processor.py:156} INFO - Started process (PID=21465) to work on /opt/airflow/dags/test.py
[2023-09-14T15:03:04.400+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:03:04.410+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:04.410+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:03:04.944+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:03:05.120+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:05.119+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:03:05.219+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:05.219+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:03:05.291+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.900 seconds
[2023-09-14T15:03:35.558+0000] {processor.py:156} INFO - Started process (PID=21525) to work on /opt/airflow/dags/test.py
[2023-09-14T15:03:35.560+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:03:35.561+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:35.561+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:03:35.891+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:03:36.071+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:36.070+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:03:36.157+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:03:36.156+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:03:36.246+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.692 seconds
[2023-09-14T15:04:06.674+0000] {processor.py:156} INFO - Started process (PID=21581) to work on /opt/airflow/dags/test.py
[2023-09-14T15:04:06.676+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:04:06.689+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:06.688+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:04:07.536+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:04:07.705+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:07.704+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:04:07.779+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:07.779+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:04:07.866+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.196 seconds
[2023-09-14T15:04:38.625+0000] {processor.py:156} INFO - Started process (PID=21641) to work on /opt/airflow/dags/test.py
[2023-09-14T15:04:38.627+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:04:38.628+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:38.628+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:04:39.641+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:04:39.899+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:39.898+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:04:39.994+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:04:39.994+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:04:40.107+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.486 seconds
[2023-09-14T15:05:10.273+0000] {processor.py:156} INFO - Started process (PID=21702) to work on /opt/airflow/dags/test.py
[2023-09-14T15:05:10.275+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:05:10.288+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:10.287+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:05:10.756+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:05:11.002+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:11.002+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:05:11.104+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:11.104+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:05:11.200+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.931 seconds
[2023-09-14T15:05:41.844+0000] {processor.py:156} INFO - Started process (PID=21758) to work on /opt/airflow/dags/test.py
[2023-09-14T15:05:41.846+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:05:41.847+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:41.847+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:05:42.377+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:05:42.549+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:42.549+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:05:42.631+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:05:42.631+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:05:42.743+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.903 seconds
[2023-09-14T15:06:12.870+0000] {processor.py:156} INFO - Started process (PID=21815) to work on /opt/airflow/dags/test.py
[2023-09-14T15:06:12.871+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:06:12.872+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:12.872+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:06:13.149+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:06:13.288+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:13.287+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:06:13.376+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:13.376+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:06:13.499+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 0.643 seconds
[2023-09-14T15:06:44.614+0000] {processor.py:156} INFO - Started process (PID=21868) to work on /opt/airflow/dags/test.py
[2023-09-14T15:06:44.627+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:06:44.638+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:44.637+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:06:45.121+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:06:45.469+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:45.468+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:06:45.612+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:06:45.612+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:06:45.718+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.113 seconds
[2023-09-14T15:07:18.965+0000] {processor.py:156} INFO - Started process (PID=21925) to work on /opt/airflow/dags/test.py
[2023-09-14T15:07:18.976+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:07:18.987+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:18.986+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:07:20.262+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:07:20.837+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:20.837+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:07:21.099+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:21.099+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:07:21.265+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.308 seconds
[2023-09-14T15:07:53.839+0000] {processor.py:156} INFO - Started process (PID=21988) to work on /opt/airflow/dags/test.py
[2023-09-14T15:07:53.846+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:07:53.859+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:53.859+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:07:55.147+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:07:55.708+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:55.707+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:07:55.967+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:07:55.967+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:07:56.166+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.352 seconds
[2023-09-14T15:08:28.358+0000] {processor.py:156} INFO - Started process (PID=22042) to work on /opt/airflow/dags/test.py
[2023-09-14T15:08:28.370+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:08:28.373+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:08:28.372+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:08:29.285+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:08:29.828+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:08:29.827+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:08:30.069+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:08:30.069+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:08:30.249+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.900 seconds
[2023-09-14T15:09:02.065+0000] {processor.py:156} INFO - Started process (PID=22099) to work on /opt/airflow/dags/test.py
[2023-09-14T15:09:02.095+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:09:02.098+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:02.098+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:09:03.228+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:09:03.798+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:03.798+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:09:04.045+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:04.045+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:09:04.220+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.187 seconds
[2023-09-14T15:09:34.502+0000] {processor.py:156} INFO - Started process (PID=22157) to work on /opt/airflow/dags/test.py
[2023-09-14T15:09:34.535+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:09:34.538+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:34.537+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:09:35.140+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:09:35.844+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:35.844+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:09:36.128+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:09:36.127+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:09:36.307+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.824 seconds
[2023-09-14T15:10:06.911+0000] {processor.py:156} INFO - Started process (PID=22209) to work on /opt/airflow/dags/test.py
[2023-09-14T15:10:06.928+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:10:06.931+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:06.931+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:10:08.159+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:10:08.762+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:08.762+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:10:09.016+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:09.004+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:10:09.202+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.319 seconds
[2023-09-14T15:10:39.626+0000] {processor.py:156} INFO - Started process (PID=22266) to work on /opt/airflow/dags/test.py
[2023-09-14T15:10:39.642+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:10:39.644+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:39.644+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:10:41.126+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:10:42.099+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:42.098+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:10:42.389+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:10:42.389+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:10:42.571+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.955 seconds
[2023-09-14T15:11:13.767+0000] {processor.py:156} INFO - Started process (PID=22324) to work on /opt/airflow/dags/test.py
[2023-09-14T15:11:13.775+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:11:13.782+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:13.778+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:11:15.230+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:11:15.857+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:15.856+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:11:16.164+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:16.151+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:11:16.317+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 2.569 seconds
[2023-09-14T15:11:47.732+0000] {processor.py:156} INFO - Started process (PID=22383) to work on /opt/airflow/dags/test.py
[2023-09-14T15:11:47.735+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/test.py for tasks to queue
[2023-09-14T15:11:47.748+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:47.748+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/test.py
[2023-09-14T15:11:48.546+0000] {processor.py:809} INFO - DAG(s) dict_keys(['keren']) retrieved from /opt/airflow/dags/test.py
[2023-09-14T15:11:49.067+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:49.066+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-09-14T15:11:49.332+0000] {logging_mixin.py:149} INFO - [2023-09-14T15:11:49.331+0000] {dag.py:3490} INFO - Setting next_dagrun for keren to 2023-09-14T16:00:00+00:00, run_after=2023-09-14T17:00:00+00:00
[2023-09-14T15:11:49.542+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/test.py took 1.836 seconds
